client/src/components/onboarding/UserAvailability.js

```diff
@@ -47,11 +48,49 @@ for (let i = 0; i <= 23; i++) {
     );
 }
 
+function CheckBox({days,handler}) {
 
-function CheckBox() {
+    const classes = useStyles();
+
+    return (
+        <FormGroup row>
+            {days.map((date,index)=>(
+                <FormControlLabel
+                key={date.id}
+                value={index}
+                checked={date.state}
+                control={<Checkbox  color="primary"/>}
+                label={<Typography variant="subtitle2" className={classes.checkBoxText}>{date.value}</Typography>}
+                labelPlacement="bottom"
+                onChange={handler}
+                classes={{root:classes.checkBox}}
+                style={{color:date.color}}
+                />
+            ))}
+        </FormGroup>
+    )
+}
+
+function UserAvailability({setters}) {
 
     const classes = useStyles();
 
+    const alertContext = useContext(AlertContext)
+
+    const userContext = useContext(UserContext)
+    
+    const [startTime,setStartTime] = React.useState(9)
+
+    const [endTime,setEndTime] = React.useState(17)
```

Comment:
I think it will be better to import `{useState}` and use `useState` here instead `Ract.useState()`